phc/session.py

```diff
@@ -1,19 +1,25 @@
 """A Python Module for managing PHC sessions."""
 
 import os
-import jwt
 import time
 from urllib.parse import urlparse
 
+import jwt
+
+from phc.adapter import Adapter
+
 
 class Session:
     """Represents a PHC API session"""
 
+    adapter: Adapter
+
     def __init__(
         self,
         token: str = os.environ.get("PHC_ACCESS_TOKEN"),
         refresh_token: str = os.environ.get("PHC_REFRESH_TOKEN"),
         account: str = os.environ.get("PHC_ACCOUNT"),
+        adapter=Adapter(),
```

Comment:
If you create the Adapter as a default argument, then every Session will share the same Adapter instance. Is that OK? That Python behavior is usually a surprise (only evaluating default args once) and I tend to assign `None` as a default arg and then check for None in the code for anything potentially mutable.

FWIW, I'm pretty sure those calls to `os.environ.get` will only be evaluated once also, so if someone creates a `Session`, then updates those env vars and tries to create another `Session`, the updated values will not be used. That might issues I've had trying to use those. I found I had to restart my kernel and set them before importing or calling the SDK for them to be honored.

Reply:
Totally true. I can adjust that.